modules/solvers/voltage-divider.stanza

```diff
@@ -25,23 +25,27 @@ public defstruct VoltageDividerSolution :
   vo: Toleranced
 
 public defn voltage-divider-solver (v-in: Toleranced,
-                             v-out: Toleranced,
-                             current: Double) -> VoltageDividerSolution|False:
+                                    v-out: Toleranced,
+                                    current: Double) -> VoltageDividerSolution|False:
   ; Look for highest acceptable tolerance in EIA standard values: [20% 10% 5% 2% 1% 0.5% 0.25% 0.1%]
   val current-tolerance = 0.1
-  val goal-r-hi = (typ-value(v-in) - typ-value(v-out)) / current
-  val goal-r-lo = typ-value(v-out) / current
+  val goal-r-hi = (v-in - v-out) / current
+  val goal-r-lo = v-out / current
```

Comment:
I assume that the arithmetic operators now handle the typ-value() extraction?

Reply:
Before:
`goal-r-xx` variables were Doubles. The operation was divide(Double, Double).

PR:
`goal-r-xx` variables are Toleranced. The operation is divide(Toleranced, Double).

`typ-value(v-out) / current` is now a subset of the operations that happen when calculating the `goal-r-xx`.

Reply:
I reverted this change. This PR is not intended to change the solution space, and changing the goals from Double to Toleranced was affecting that.